src/Contracts/SearchQuery.php

```diff
@@ -429,6 +436,29 @@ public function setAttributesToSearchOn(array $attributesToSearchOn): self
         return $this;
     }
 
+    public function setRetrieveVectors(?bool $retrieveVectors): self
+    {
+        $this->retrieveVectors = $retrieveVectors;
+
+        return $this;
+    }
+
+    /**
+     * This is an EXPERIMENTAL feature, which may break without a major version.
+     * It's available from Meilisearch v1.16.
+     * To enable it properly and use multimodal search, it's required to activate it through the /experimental-features route.
+     *
+     *  More info: https://www.meilisearch.com/docs/reference/api/experimental-features
+     *
+     * @param array<string, mixed>|null $media
```

Comment:
to me mixed looks to broad. also i failed to find any info about this media

Reply:
Yep, same 🙁 Maybe @Strift do you have more information on how the `media`'s param is supposed to be structured?